happy/HappyInternet.py

```diff
@@ -86,12 +86,13 @@ def __init__(self, opts=options):
         self.seed = int(opts["seed"])
         self.prefix = '172.16.' + opts["seed"] + '.'
 
-        if "weave_service_address" in os.environ.keys():
-            tier = os.environ['weave_service_address'].split(".")[3][0:3]
-            self.isp_id = tier + self.isp_id
-        else:
-            tier = "test"
-            self.isp_id = tier + self.isp_id
+        if self.add:
```

Comment:
As far as I can see the only net-new line here is 89: 'if self.add:', correct?

What is the before / after behavior? What was working and what is fixed by the change?

Reply:
the reason to have self.add flag is: during we add happy-internet, we save isp_id in .happy_state.json already where you had question for line #328 and #329.
so we don't need to build self.isp_id any more in happy-internet delete case.